pkgs/development/libraries/libpfm/default.nix

```diff
@@ -9,11 +17,13 @@ stdenv.mkDerivation rec {
     sha256 = "0jabhjx77yppr7x38bkfww6n2a480gj62rw0qp7prhdmg19mf766";
   };
 
-  installFlags = "DESTDIR=\${out} PREFIX= LDCONFIG=true";
+  makeFlags = lib.optional (arch != null) "ARCH=${arch}";
```

Comment:
You should be able to use stdenv.hostPlatform.parsed.cpu.name here.

Reply:
Oh I see that now, I misunderstood how the `ARCH` logic worked.